Extensions/xkit_preferences.js

```diff
@@ -1006,7 +1006,7 @@ XKit.extensions.xkit_preferences = new Object({
 
 		if (obj.name.startsWith("xkit_") && XKit.tools.get_setting("xkit_show_internals","false") === "false") { return ""; }
 
-		var blacklisted_extensions = ["xkit_installer"];
+		var blacklisted_extensions = ["xkit_installer","old_stats"];
```

Comment:
Removing extensions without telling people is a big no, IMO. Let's have old_stats in until we spring clean next time (which should be color quotes, old stats and maybe also stuff like fixmyposts (Could we "update" fixmyposts to install RDO maybe?)